cmd/request/request_test.go

```diff
@@ -119,3 +119,48 @@ func TestRequestCmd_Execute_MissingRequiredServiceFlag(t *testing.T) {
 	err := testutils_cobra.ExecutePingcli(t, "request", fmt.Sprintf("environments/%s/populations", os.Getenv(options.PingOneAuthenticationWorkerEnvironmentIDOption.EnvVar)))
 	testutils.CheckExpectedError(t, err, &expectedErrorPattern)
 }
+
+// Test Request Command with Header Flag
+func TestRequestCmd_Execute_HeaderFlag(t *testing.T) {
+	err := testutils_cobra.ExecutePingcli(t, "request",
+		"--"+options.RequestServiceOption.CobraParamName, "pingone",
+		"--"+options.RequestHTTPMethodOption.CobraParamName, "GET",
+		"--"+options.RequestHeaderOption.CobraParamName, "Content-Type: application/vnd.pingidentity.user.import+json",
+		fmt.Sprintf("environments/%s/users", os.Getenv(options.PingOneAuthenticationWorkerEnvironmentIDOption.EnvVar)),
+	)
+	testutils.CheckExpectedError(t, err, nil)
+}
+
+// Test Request Command with Header Flag with and without spacing
+func TestRequestCmd_Execute_HeaderFlagSpacing(t *testing.T) {
+	err := testutils_cobra.ExecutePingcli(t, "request",
+		"--"+options.RequestServiceOption.CobraParamName, "pingone",
+		"--"+options.RequestHTTPMethodOption.CobraParamName, "GET",
+		"--"+options.RequestHeaderOption.CobraParamName, "Test-Header:TestValue",
+		"--"+options.RequestHeaderOption.CobraParamName, "Test-Header:\tTestValue",
```

Comment:
Update key to different key to also test multiple headers at once.
Do we want to warn if the same key is supplied twice?